### PR TITLE
GL1.3 clipping support

### PR DESCRIPTION
**Pull request by gwaldron**
Integrate new osgEarth HorizonClipPlane to support GL1.3+ clipping (replaced old ClipToGeocentricHorizon object); also a couple of implicit ref_ptr fixes in there.

### CODE REVIEW

File: SDK/simVis/SceneManager.cpp [hunk 1]
```diff
@@ -39,6 +38,7 @@
 #include "osgEarthDrivers/engine_rex/RexTerrainEngineOptions"
 #include "osgEarthDrivers/engine_mp/MPTerrainEngineOptions"
 #include "osgEarthUtil/LODBlending"
+#include "osgEarthUtil/HorizonClipPlane"
```
Comment (emminizer) [inline SDK/simVis/SceneManager.cpp]: Requires a newer osgEarth than 2.9.  Requires a if/ifdef to build properly

Comment (emminizer) [reply SDK/simVis/SceneManager.cpp]: I think the following is needed here, and at the if-0 below:
```
#include "osgEarth/Version"

#if OSGEARTH_MIN_VERSION_REQUIRED(2,10,0)
#include "osgEarthUtil/HorizonClipPlane"
#endif
...
```

File: SDK/simVis/SceneManager.cpp [hunk 2]
```diff
@@ -222,7 +222,8 @@ void SceneManager::init_()
 #endif
 
   setName("simVis::SceneManager");
-
+  
+#if 0
```
Comment (emminizer) [inline SDK/simVis/SceneManager.cpp]: #if OSGEARTH_VERSION_LESS_THAN(2,10,0))

Comment (gwaldron) [reply SDK/simVis/SceneManager.cpp]: PR updated - please re-evaluate

**Pull request merged**